src/enter.js

```diff
@@ -26,6 +26,6 @@ export default class Enter extends Feature {
 		this.listenTo( editingView, 'enter', ( evt, data ) => {
 			editor.execute( 'enter' );
 			data.preventDefault();
-		} );
+		}, 'low' );
```

Comment:
I mean that if you add a listener with `low` priority after (in code) discussed listener, the new listener will be fired as a second one. So `lowest` is not the only option to go.